blockchain/txvalidator.go

```diff
@@ -2160,6 +2160,42 @@ func (b *BlockChain) checkCRCProposalTransaction(txn *Transaction,
 		return errors.New("budgets exceeded the maximum limit")
 	}
 
+	if proposal.ProposalType == payload.ChangeProposalOwner {
+		if !b.crCommittee.IsInElectionPeriod() || b.crCommittee.IsInVotingPeriod(blockHeight) {
+			return errors.New("cr proposal tx must be at election Period and not during voting period")
+		}
+
+		proposalState := b.crCommittee.GetProposal(proposal.PreviousHash)
+		if proposalState.Status != crstate.VoterAgreed {
+			return errors.New("proposal status is not VoterAgreed")
+		}
+
+		if b.crCommittee.ExistProposal(proposal.Hash()) {
```

Comment:
current proposal is not exist at the moment , we don't need this if statement i think.